packages/oas/src/operation/index.ts

```diff
@@ -153,6 +153,11 @@ export class Operation {
     return matchesMimeType.xml(this.getContentType());
   }
 
+  // Checks if the current operation is a webhook or not 
```

Comment:
```suggestion
  /**
   * Checks if the current operation is a webhook or not.
   *
   */
```

Reply:
updated comment format and added to readme!